doc/06-Demonstration.md

```diff
@@ -1,30 +1,31 @@
 # <a id="Demonstration"></a>Demonstration
```

Comment:
#287 will remove the Dockerfile. Please revert the changes made here as this file is also subject to removal.

Reply:
That issue shouldn’t take long to resolve. I'll rebase this one on top if necessary.

Reply:
Ready to rebase.